code/modules/recycling/conveyor.dm

```diff
@@ -390,10 +400,18 @@ GLOBAL_LIST_EMPTY(conveyors_by_id)
 	update_linked_switches()
 
 
-/obj/machinery/conveyor_switch/attackby(obj/item/attacking_item, mob/user, params)
-	if(is_wire_tool(attacking_item))
-		wires.interact(user)
-		return TRUE
+/obj/machinery/conveyor_switch/wirecutter_act(mob/living/user, obj/item/I)
+	wires.interact(user)
+	return TRUE
+
+/obj/machinery/conveyor_switch/multitool_act(mob/living/user, obj/item/I)
+	var/speed = tgui_input_number(user, "Set the speed of the conveyor belts", "Speed", id, 20, 0.2)
+	if(!speed || QDELETED(user) || QDELETED(src) || !usr.canUseTopic(src, BE_CLOSE, FALSE, NO_TK))
+		return
+	conveyor_speed = speed
```

Comment:
it seems a bit strange to store speed in seconds rather then deciseconds, though I suppose it would make the defaults easier